ParisBNF/abb/BNFabb111.xml

```diff
@@ -0,0 +1,689 @@
+                                      <locus target="#44r"/>
+                                       <desc type="StampExlibris">Stamp of the library.</desc>
+                                    </item>
+
+                                    </list>
+                                    </additions>
+
+
+
+                                </physDesc>
+                                    <history>
+                                       <origin>
+                                          <origDate notBefore="1400" notAfter="1499">15th century.</origDate>
+                                          The owner of the manuscript <ref target="#p1"/> was <persName ref="PRS13365Amlakawi" role="owner">ʾAmlākāwi</persName>,
+                                           mentioned in the concluding formulas, on <ref target="#66vb"/>,
+                                          <ref target="#76vb"/>, <ref target="#77rb"/>, <ref target="#81r"/>
+                                           and many others. His name has been added
+                                          in a secondary hand, in some places obviously instead of another
+                                          name that was erased (e.g. <ref target="#84ra"/>,
+                                          <ref target="#85ra"/>, <ref target="#87ra"/>). At a few places, the name of
+                                          <persName ref="PRS13365Amlakawi" role="owner">ʾAmlākāwi</persName> was overwritten with
+                                          still another name, <persName ref="PRS13366Aragawi" role="owner" cert="medium">ʾAragāwi</persName>
+                                          (<ref target="#43rb"/>,
+                                          <ref target="#43vb"/>, <ref target="#44ra"/>,
+                                          <ref target="#59ra"/>).
+                                        </origin>
+                                    </history>
+                                    </msPart>
+
+
+                                    <msPart xml:id="p2">
+                                      <msIdentifier/>
+                                      <msContents>
+                                        <summary/>
+                             <msItem xml:id="p2_i1">
+
+                               <locus from="103ra" to="110rb"/>
+                               <title type="complete" ref="LIT1385Fekkar"></title>
+                               <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡</hi> አሐዱ፡
+                              አምላክ። ወዝንቱ፡ ነገር፡ ወፍካሪ፡ ኢየሱስ፡ ወተስእልዎ፡ አርዳኢሁ፡ ወእንዘ፡ ይመስሐ፡ ምስለሆሙ፡ ወሐተትዎ፡
+                            ወይቤልዎ፡ እስኩ፡ ንግረኒ፡ ኦሊቅነ፡ በደኅሪ፡ ትውልድ፡ ዘይመጽእ፡ መዋዕል፡ ምንተ፡ ይከውን፡ ተእምርቱ። ወአውሦሙ፡
+                               </incipit>
+                               <note>Cp. <bibl>
+                                  <ptr target="bm:Nosnitsin2005EAEFekkareI"/></bibl>
```

Comment:
is the bibliography for the ms or for the work?

Reply:
it mentions the ms.